api/registry/test/test_passport_get_score.py

```diff
@@ -132,6 +134,55 @@ def test_get_single_score_for_address(
             == passport_holder_addresses[0]["address"].lower()
         )
 
+    def test_get_single_score_for_address_in_path(
+        self,
+        scorer_api_key,
+        passport_holder_addresses,
+        scorer_community,
+        paginated_scores,
+    ):
+        client = Client()
+        response = client.get(
+            f"/registry/score/{scorer_community.id}/{passport_holder_addresses[0]['address']}",
+            HTTP_AUTHORIZATION="Token " + scorer_api_key,
+        )
+        assert response.status_code == 200
+
+        assert (
+            response.json()["address"]
+            == passport_holder_addresses[0]["address"].lower()
+        )
+
+    def test_get_single_score_for_address_in_path_for_other_community(
```

Comment:
Change test name to: `test_cannot_get_single_score_for_address_in_path_for_other_community`